using-mobile-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## February 4, 2020
+
+**Android Identity 1.2.0** 
+
+- Added support for the optional `AdobeCallbackWithError` available in Android Core 1.5.0 on the following APIs: `appendVisitorInfoForURL`, `getUrlVariables`, `getIdentifiers`, `getExperienceCloudId`. When the AdobeCallbackWithError is used, a default 500ms timeout will be enforced when retrieving the Mobile SDK values; an AdobeError will be returned if the operation timed out or it was not successful.
```

Comment:
"When the AdobeCallbackWithError is used, a default 500ms timeout is enforced when retrieving the Mobile SDK values; an AdobeError is returned if the operation times out or is not successful."